girder/girder_large_image/rest/large_image_resource.py

```diff
@@ -59,7 +59,7 @@ def createThumbnailsJobTask(item, spec):
         except TileGeneralError as exc:
             status['failed'] += 1
             status['lastFailed'] = str(item['_id'])
-            logger.info('Failed to get thumbnail for item %s: %r' % (item['_id'], exc))
+            logger.error('Failed to get thumbnail for item %s: %r' % (item['_id'], exc))
```

Comment:
I think when `imageKey` is present, this should actually do nothing in the exception rather than increment failed and log anything.